entries/k/kakao.com.json

```
@@ -0,0 +1,25 @@
+{
+    "Kakao": {
+      "domain": "kakao.com",
+      "tfa": [
+        "sms",
+        "custom-software"
+      ],
+      "custom-software": [
+        "KakaoTalk app"
+      ],
+      "documentation": "https://cs.kakao.com/helps?service=52&category=1095&locale=en",
+      "recovery": "https://cs.kakao.com/helps_html/1073210250?locale=en",
```

Since the documentation page explains how to set up 2FA and recover an account with 2FA, only the documentation attribute is needed.
```suggestion
      "documentation": "https://cs.kakao.com/helps?service=52&category=1095",
```